algosdk/atomic_transaction_composer.py

```diff
@@ -479,6 +479,55 @@ def submit(self, client: algod.AlgodClient) -> List[str]:
         self.status = AtomicTransactionComposerStatus.SUBMITTED
         return self.tx_ids
 
+    def simulate(
+        self, client: algod.AlgodClient
+    ) -> "SimulateAtomicTransactionResponse":
+
+        if self.status <= AtomicTransactionComposerStatus.SUBMITTED:
+            self.gather_signatures()
+        else:
+            raise error.AtomicTransactionComposerError(
+                "AtomicTransactionComposerStatus must be submitted or "
+                "lower to simulate a group"
+            )
+
+        simulation_result: Dict[str, Any] = client.simulate_transactions(
+            self.signed_txns, response_format="json"
```

Comment:
I *think* `json` is the default for every algod call in this SDK so we don't need to define this: https://github.com/algorand/py-algorand-sdk/blob/882052efa331ceafa5d0afd695bb8c8a426efc6d/algosdk/v2client/algod.py#L39